frontend/src/component/pages/shared/UserShortLinksSection.spec.tsx

```diff
@@ -152,23 +148,82 @@ describe('UserShortLinksSection component', () => {
     expect(onPageLoad).toBeCalledWith(0, pageSize);
   });
 
-  test('should load correct short links when page changed', () => {
+  test('should load short links for initial page correctly', () => {
     const pageSize = 3;
-    let pagedShortLinks: IPagedShortLinks = {
+    const initialPagedShortLinks: IPagedShortLinks = {
       shortLinks: [
         { originalUrl: 'https://longurl.com/1', alias: 'alias1' },
         { originalUrl: 'https://longurl.com/2', alias: 'alias2' },
         { originalUrl: 'https://longurl.com/3', alias: 'alias3' }
       ],
       totalCount: 12
     };
-    const onPageLoad = jest.fn();
 
-    const { container } = render(
+    const onPageLoad = jest.fn((offset: number, pageSize: number) => {
+      setTimeout(() => {
+        rerender(
+          <UserShortLinksSection
+            onPageLoad={jest.fn}
+            pageSize={pageSize}
+            pagedShortLinks={initialPagedShortLinks}
+          />
+        );
+      }, 10);
+    });
+
+    const { container, rerender } = render(
```

Comment:
I recommend to explicitly define `rerender` before line 162 for clarity.

Reply:
@byliuyang , `rerender` is method returned from render method.
You wanna do this?
```
let rerender: any;
// code for onPageLoad

const result = render(
  <UserShortLinksSection onPageLoad={onPageLoad} pageSize={pageSize} />
);
const container = result.container;
rerender = result.rerender;
```

Reply:
@rohithbalaji123 Sounds good to me. I mean we don't have much other choices...